integration-tests/src/test/java/uk/gov/di/authentication/api/LogoutIntegrationTest.java

```diff
@@ -88,8 +88,7 @@ private AuthorizationRequest generateAuthRequest() {
     }
 
     private String buildCookieString(String sessionID, String clientSessionID) {
-        return format(
-                "%s=%s.%s; Max-Age=%d; %s",
-                "gs", sessionID, clientSessionID, 1800, "Secure; HttpOnly;");
+        var cookie = new HttpCookie("gs", sessionID + "." + clientSessionID);
+        return cookie.toString();
```

Comment:
Looks good. I thought we were going to keep this one as is because `cookie.toString()` doesn't add e.g. Max-Age?

Reply:
I think this is fine to be cookie.toString as it will replicate a client sending a request to a server which wouldn't included max age